src/chart/Utils.ts

```diff
@@ -76,45 +76,6 @@ export const flatten = data =>
         return [...acc, item]
     }, []);
 
-/**
```

Comment:
Where's is this located now?

Reply:
For some reason, If I checkout the branch I see the file src/chart/ChartUtils but I don't see it on the PR tho